src/assets/stylesheets/screens/_sign-in.scss

```diff
@@ -0,0 +1,26 @@
+.container-sign-in {
+  display: flex;
+  flex-direction: column;
+  width: rem(320px);
+}
+
+.form-input-group {
+  display: flex;
+  flex-direction: column;
+  width: rem(320px);
```

Comment:
@gutakk Do we need to set this while the container already has the fixed width?

Reply:
You're right. Fixed in 272c1662fb02cd83f73cdd6e01d246f0adfa2117